gateway/gateway_test.go

```diff
@@ -114,12 +114,8 @@ func (c *testContext) Setup() {
 	c.mockApp = mocksApp.NewMockApp(c.mockCtrl)
 	c.mockRateLimiter = mocksRateLimiter.NewMockRateLimiter(c.mockCtrl)
 
-	tFunc := c.asyncHelper.ExpectAndNotifyCallbackWithName("process_config")
```

Comment:
might need to reinstate this functionality

Reply:
What was this used for?

Reply:
verifies that the `tFunc` has been called